nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Tag Index & Admin Tools button

# Description
- added tag index view
- all tags listed in alphabetical order
- added admin tools button to navbar w/ dropdown menu for tag management and category management

[-ticket:](https://github.com/nss-day-cohort-41/tabloidmvc-the-gefilte-fish/issues/13)

## Type of change
# Testing Instructions
- clone repository
- navigate to root folder (of repo)
- run included sql scripts to create and populate database
- dotnet run
- navigate to the port that your system is running the build on in Google Chrome
- login 'admin@example.com'
- observe admin tools button in top right corner
- on click dropdown menu appears
- tag management
- user will be taken to tag index

# Checklist:
